tests/providers/docker/operators/test_docker.py

```diff
@@ -510,6 +511,26 @@ def test_execute_unicode_logs(self):
             logging.raiseExceptions = original_raise_exceptions
             print_exception_mock.assert_not_called()
 
+    @parameterized.expand(
+        [
+            (None, 99, AirflowSkipException),
+            ({"skip_exit_code": 100}, 100, AirflowSkipException),
+            ({"skip_exit_code": 100}, 101, AirflowException),
+            ({"skip_exit_code": None}, 99, AirflowException),
+        ]
+    )
+    def test_skip(self, extra_kwargs, actual_exit_code, expected_exc):
```

Comment:
Would be nice also add this kind of test to docker decorator
https://github.com/apache/airflow/blob/4f91931b359f76ae38272c727bfe21a18a470f2b/tests/providers/docker/decorators/test_docker.py#L26

Right not docker decorator tests run under non-mocked environment